lib/features/email/data/datasource_impl/email_hive_cache_datasource_impl.dart

```diff
@@ -149,4 +154,36 @@ class EmailHiveCacheDataSourceImpl extends EmailDataSource {
       return await _emailCacheManager.storeEmail(accountId, session.username, email.toEmailCache());
     }).catchError(_exceptionThrower.throwException);
   }
+
+  @override
+  Future<void> storeOpenedEmail(Session session, AccountId accountId, DetailedEmail detailedEmail) {
+    return Future.sync(() async {
+      final task = HiveTask(
+          runnable: () async {
+
+            final detailedEmailExisted = await _openedEmailCacheManager.openedDetailsEmailCacheHit(accountId, session.username,detailedEmail);
+            log('EmailHiveCacheDataSourceImpl::storeOpenedEmailToCache():isDuplicateDetailedEmail: $detailedEmailExisted');
+
+            if (detailedEmailExisted) {
+              return Future.value();
+            }
+
+            final fileSaved = await _fileUtils.saveToFile(
+              nameFile: detailedEmail.emailId.asString,
+              content: detailedEmail.htmlEmailContent ?? '',
+              folderPath: detailedEmail.folderPath
+            );
+
+            log('EmailHiveCacheDataSourceImpl::storeOpenedEmailToCache():fileSavedPath: ${fileSaved.path}');
```

Comment:
rename to `log('EmailHiveCacheDataSourceImpl::storeOpenedEmail():fileSavedPath: ${fileSaved.path}');`